### PR TITLE
feat: add User API documentation

### PR DESCRIPTION
**Pull request by magajh**
This PR adds the swagger documentation for the user API

https://openedx.je.edunextstage.net/eox-core/api-docs/

### CODE REVIEW

File: eox_core/api/v1/views.py [hunk 1]
```diff
@@ -159,35 +328,154 @@ def get(self, request, *args, **kwargs):
...
+        The username used to identify the user.
+
+    - `password` (**optional**, string, _body_):
+        The password of the new user. If `skip_password` is True, this field would be omitted.
+
+    - `fullname` (**optional**, string, _body_):
+        The full name to be assigned.
+
+    - `is_active` (boolean, _body_):
+        Flag indicating whether the enrollment is active.
```
Comment (mariajgrimaldi) [inline eox_core/api/v1/views.py:356]: the enrollment or the user?

Comment (magajh) [reply eox_core/api/v1/views.py]: thank you. Done

**Comment by magajh** [comment]
@MoisesGSalas

**Review by MoisesGSalas — APPROVED** [review]
LGTM. nitpick: If you can, fix the examples in the docstrings, you are mixing john and jane, etc.

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**